Makefile

```diff
@@ -291,6 +291,21 @@ install_targets-y += install_libplatsbi
 install_targets-y += install_firmwares
 endif
 
+# Convenient "make run" command for emulated platforms
+.PHONY: run
+run: all
+ifneq ($(platform-runcmd),)
+	$(platform-runcmd)
```

Comment:
How about having RUN_ARGS command-line parameter to pass additional args to $(platform-runcmd).

May be just replace "$(platform-runcmd)" with "$(platform-runcmd) $(RUN_ARGS)". Suggestions??